deploy/infrabox/templates/dockerd-config-map.yaml

```diff
@@ -6,3 +6,8 @@ metadata:
 data:
   daemon.json: |-
 {{ required "job.docker_daemon_config is required" .Values.job.docker_daemon_config | indent 4 }}
+  {{- if empty .Values.job.rootca_pem }}
```

Comment:
if root ca is not empty, then mount root ca

Reply:
All if-else improved with the not empty judgement